README.md

```diff
@@ -97,7 +97,8 @@ https://codesandbox.io/s/ynn88nx9x?view=split
 The returned HTML snippet from the Twitter transformer will only be
 automatically recognized as an [Embedded Tweet][embedded-tweet-docs] when
 [Twitter's widget JavaScript][twitter-widget-javascript-docs] is included on the
-page.  
+page.
+
```

Comment:
I added another line-break to make Prettier happy as it was removing the breaks in the git hook. I can recommit this with --no-verify if we prefer the original formatting.

Reply:
Their was a double space (`  `) after the point (`.`), which is a valid `<br />` in Markdown syntax, so please put this one back.